src/diagrams/class/parser/classDiagram.jison

```diff
@@ -29,8 +37,7 @@
 
 "class"               return 'CLASS';
 "cssClass"            return 'CSSCLASS';
-"callback"            return 'CALLBACK';
-"link"                return 'LINK';
+"click"               return 'CLICK';
```

Comment:
Thank you for this part.  I was originally having issues with getting this to work correctly, which is why I separated them - and always intended to go back and fix it but never got around to it.